src/transformers/models/roformer/configuration_roformer.py

```diff
@@ -0,0 +1,126 @@
+# coding=utf-8
+# Copyright 2021 junnyu and The HuggingFace Inc. team. All rights reserved.
```

Comment:
```suggestion
# Copyright 2021 The HuggingFace Inc. team. All rights reserved.
```
Copyright is usually attributed to Companies, not individuals (I think this came automatically from the templates)